app/controllers/api/alert_definition_profiles_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  class AlertDefinitionProfilesController < BaseController
+    include Subcollections::AlertDefinitions
+
+    REQUIRED_FIELDS = %w(description mode).freeze
+
+    def create_resource(type, id, data = {})
+      assert_id_not_specified(data, type)
```

Comment:
Safe to remove this as it is called in api::base_controller:generic (I checked :beer:)